terraform/modules/app-ecs-albs/main.tf

```diff
@@ -279,14 +287,97 @@ resource "aws_route53_record" "alerts_private_record" {
   }
 }
 
+######################################################################
+# ----- prometheus public ALB -------
+######################################################################
+
+# AWS should manage the certificate renewal automatically
+# https://docs.aws.amazon.com/acm/latest/userguide/managed-renewal.html
+# If this fails, AWS will email associated with the AWS account
+resource "aws_acm_certificate" "prometheus_cert" {
```

Comment:
So I see that you have introduced a different cert for prom and a different cert for alertmanager. Could you explain your rough reasoning please why this is beneficial as I don't have any context of why we want to do this?

Reply:
That’s a good question. My thinking is a bit vague but the following factors contributed to the decision:

- principle of least authority: the alermanager ALB doesn’t need a prom-1 cert, so it shouldn’t have one. 
- a view towards extracting common functionality later: the alert manager ALB and the Prometheus ALB have very similar functionality. I’ve been grouping the resources for each together in the app-ecs-albs module. The old certificate was awkward in that it was one of very few pieces of shared functionality which would make a future extracting of a common “ALB” module more difficult.

Reply:
Thanks, good to go with this. Will add some of this context to the commit message